custom-example/android/AndroidManifest.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0"?>
```

Comment:
I did try generating the `package` from `QGC_ORG_NAME`, but it aint all that easy and it need to agree with the folder structure where the `QGCActivity.java` lives. Since custom QGCs may have different permissions, all things considered, I think to replicate the entire AndroidManifest.xml is more honest.